protobufs/invoice/service.proto

```diff
@@ -1,11 +1,11 @@
 syntax = "proto3";
 
-package invoice;
+package inv;
 
-option go_package = "invoicepb";
+option go_package = "invpb";
```

Comment:
can we add the key `api` or `client` somewhere instead?

Reply:
that was the first thing I tried, its just too long.. and we are removing these soon anyway